File: spring-email-java/src/main/java/com/github/migangqui/spring/email/bean/Email.java
package com.github.migangqui.spring.email.bean;

import java.io.InputStream;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@Builder
public class Email {
	
	private String from;
	private String to;
	private String subject;
	private String body;
	private InputStream file; 
	private String filename;
	
}

File: spring-email-kotlin/src/main/kotlin/com/github/migangqui/spring/email/bean/Email.kt
package com.github.migangqui.spring.email.bean

import java.io.InputStream

data class Email(
    val from: String,
    val to: String,
    val subject: String? = null,
    val body: String,
    val file: InputStream? = null,
    val filename: String? = null
)